Marlin/Conditionals_LCD.h

```diff
@@ -101,7 +110,8 @@
   #if ENABLED(ULTIMAKERCONTROLLER)              \
    || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) \
    || ENABLED(G3D_PANEL)                        \
-   || ENABLED(RIGIDBOT_PANEL)
+   || ENABLED(RIGIDBOT_PANEL)                   \
+   || ENABLED(REPRAPWORLD_KEYPAD)
```

Comment:
Checking the `ultralcd.cpp` code, I see that actually the Reprapworld Keypad was always implemented in a way that it requires an LCD display (#472). Poking around the internet it looks like it has more than one display option — I've seen a 20x4 and a 20x2 display in my search.

So… Should we tweak the `REPRAPWORLD_KEYPAD` code so it can work without an LCD?